prow/prstatus/prstatus.go

```diff
@@ -341,8 +340,15 @@ func (da *DashboardAgent) GetHeadContexts(ghc githubClient, pr PullRequest) ([]C
 }
 
 // GetUser attempts to get the currently authenticated Github user.
-func (da *DashboardAgent) GetUser(client *ghclient.Client) (*gogithub.User, error) {
-	return client.GetUser("")
+func (da *DashboardAgent) GetUser(client *gogithub.Client) (*gogithub.User, error) {
```

Comment:
We should just change the code to use the BotName() function directly

Reply:
This should do the trick, did you come up with something similar @hpandeycodeit? 

https://github.com/kubernetes/test-infra/compare/master...midnightconman:prstatus-new-githubclient-20190225141936?expand=1

Reply:
yes I am in process of making a commit :)